src/FlashList.tsx

```diff
@@ -0,0 +1,121 @@
+import { FlashList as SPFlashList, FlashListProps } from '@shopify/flash-list'
+import React from 'react'
+
+import { AnimatedFlashList } from './helpers'
```

Comment:
This also needs to be updated.

Reply:
I would suggest removing `AnimatedFlashList`  from `./helpers` in this case and just colocating it in the same file.

Reply:
Yes, Just not do the same thing twice